src/service/proximity-scoring-service.js

```diff
@@ -154,7 +166,9 @@ class ProximityScoringService {
             normalizedDistance = normalizedDistance.mod(UINT64_MAX_BN.add(1));
         }
 
-        let normalizedStake = stakeScaleFactor.mul(stake - minStake).div(maxStake - minStake);
+        let normalizedStake = stakeScaleFactor
```

Comment:
This is not correct, we expect stake to be number, so should be minStake and maxStake

Reply:
Nvm, it's fine